src/dev/i18n/README.md

```diff
@@ -0,0 +1,175 @@
+# I18n Build Tools
+
+## Default messages extraction tool
+
+### Description
+
+The tool is used to extract default messages from all `*.{js, ts, jsx, tsx, html, handlebars, hbs, pug}` files in provided plugins directories to a JSON file.\
+The tool uses Babel to parse code and build an AST for each file or JS expression if whole file parsing is impossible. So only static parsing is available for messages extraction. It means that no variables, function calls or dynamic expressions can be used for ids, messages and description, only strings.
```

Comment:
__nit:__ maybe remove `\` and just add a new line before `The tool uses ...`, since `The tool` is used in the first sentence you can just use `It` instead.

__nit:__ `or JS expression` ---> `or a single JS expression`.

__nit:__ `So only static parsing is available for messages extraction. It means that no variables, function calls or dynamic expressions can be used for ids, messages and description, only strings.` Maybe something like this would be clearer? `The tool is able to validate, extract and match IDs, default messages and descriptions only if they are defined statically and together, otherwise it will fail with detailed explanation. That means one can't define ID in one place and default message in another, or use function call to dynamically create default message etc.`